packages/fluentui/docs/src/examples/components/Card/BestPractices/CardBestPractices.tsx

```diff
@@ -5,9 +5,16 @@ import ComponentBestPractices from '../../../../components/ComponentBestPractice
 const doList = [
   'Limit use of `aria-roledescription` property to clarifying the purpose of card, or to providing a more specific description of a widget.',
   'Use `cardBehavior` for disabled cards, so it would not be focusable.',
+  'Use `cardsContainerBehavior` for cards container, so it would be navigable using arrows.',
+  'Use `cardChildrenFocusableBehavior` for focusable cards with inteactive children.',
+  'Use `cardFocusableBehavior` for focusable cards without inteactive children.',
+  'Use `cardSelectableBehavior` for selectable cards without inteactive children.',
 ];
 
-const dontList = ['Use `onClick` property together with clickable children.'];
+const dontList = [
+  'Use `onClick` property together with clickable children.',
+  'Create selectable cards with a single text child that has `id`, that might cause incorrect narration in JAWS with virtual cursor navigation',
```

Comment:
nit: two sentences might be better: 'Create selectable cards with a single text child that has `id`. It might cause incorrect narration in JAWS with virtual cursor navigation'